sensor/common/sensor/central_sender_impl.go

```diff
@@ -135,7 +135,8 @@ func (s *centralSenderImpl) send(stream central.SensorService_CommunicateClient,
 			}
 
 			if err := wrappedStream.Send(msg.MsgFromSensor); err != nil {
-				log.Errorf("unable to send to stream: %s", err)
+				log.Errorf("unable to send to stream: %s. "+
```

Comment:
Do we potentially leak sensitive information here?

Reply:
Potentially yes. It can happen that we could log the contents of configmaps and secrets.
This is not ideal, but I am open for alternatives.

Reply:
I would be against logging these by default. We could making the message content log opt-in, either by debug level log or via an explicit env variable?

Reply:
Wdyt about this [ab9d7e4](https://github.com/stackrox/stackrox/pull/15326/commits/ab9d7e4e2de7c5c63382c2166815f6cf4e178a56)?

Reply:
ping @stehessel - can we proceed with the merge or do you want to give it another look?